### ISSUE TITLE
[MapView] Warning: Failed propType: Invalid prop `overlays[0].strokeColor` of type `number` supplied to `RCTMap`, expected `string`. Check the render method of `MapView`.

### ISSUE THREAD

**Issue by af7** [facebook/react-native#5111]
I use a `strokeColor` on `MapView` and I get the following warning:

````
Warning: Failed propType: Invalid prop `overlays[0].strokeColor` of type `number` supplied to `RCTMap`, expected `string`. Check the render method of `MapView`.
````

This is the code: 

````
        let overlays = [{
            coordinates: [
                {
                    latitude: 30,
                    longitude: 20,
                },
                {
                    latitude: annotation[0].latitude,
                    longitude: annotation[0].longitude,
                },
            ],
            lineWidth: 2,
            strokeColor: 'red',
        },
            {
                coordinates: [
                    {
                        latitude: 36,
                        longitude: 24,
                    },
                    {
                        latitude: annotation[0].latitude,
                        longitude: annotation[0].longitude,
                    },
                ],
                lineWidth: 2,
                strokeColor: 'green',
            }
        ];

       return (
            <View onLayout={(e) => this.onLayoutChange(e)}>
                <MapView
                    mapType = {this.state.mapType}
                    style={[styles.map, {height: deviceHeight/2}]}
                    region={region}
                    annotations = {annotation}
                    overlays = {overlays}
                />
            </View>
      )

````

**Comment by christopherdro** [comment]
@af7 What version of RN?

**Comment by af7** [comment]
That was on v0.17, but I do not think that it was solved on v0.18. Thanks.

**Comment by af7** [comment]
UPDATE: It is OK now (I was using the .rc version), so we can close the issue.
